### PR TITLE
Update description

### PR DESCRIPTION
**Pull request by walsehgal**
Update description to increase topic relevance.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by mypil** [comment]
@JoeDavies-MSFT - PR has been copy edited and is ready for final review.  

Thank you.

cc: @kenwith

**Comment by JoeDavies-MSFT** [comment]
Please change "In this article, you'll learn about" to "Learn about". Thanks.

File: Enterprise/manage-office-365-accounts.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ search.appverid:
 - MED150
 - BCS160
 ms.assetid: 98ca5b3f-f720-4d8e-91be-fe656548a25a
-description: "Learn what tools to use to manage your Office 365 users, and how what you can use depends on how you manage user identities. "
+description: "In this article, you'll learn about what tools to use and how to manage Office 365 accounts."
```
Comment (JohanFreelancer9) [inline Enterprise/manage-office-365-accounts.md]:
```suggestion
description: "Learn about what tools to use and how to manage Office 365 accounts."
```

**Pull request closed without merging**